data_steward/tools/consolidated_achilles_report/get_all_results_html.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import gcs_utils
+import json
+import os
+import query_reports
+
+DRC_BUCKET_PATH = 'gs://%s/' % gcs_utils.get_drc_bucket()
+
+
+def get_hpo_id(p):
+    rel_path = p[len(DRC_BUCKET_PATH):]
+    return rel_path[:rel_path.index('/')]
+
+
+def transform_bq_list(uploads):
+    """
+    Get paths to all most recent report files
+
```

Comment:
Need to add the uploads parameter to the doc string.  I can guess that it is an iterable, but it would be nice to know if you expect a specific iterable.